src/Symfony/Component/Mime/Resources/bin/update_mime_types.php

```diff
@@ -10,15 +10,13 @@
  */
 
 // load new map
-$data = file_get_contents('https://svn.apache.org/repos/asf/httpd/httpd/trunk/docs/conf/mime.types');
+$data = json_decode(file_get_contents('https://cdn.jsdelivr.net/gh/jshttp/mime-db@v1.44.0/db.json'), true);
 $new = [];
-foreach (explode("\n", $data) as $line) {
-    if (!$line || '#' == $line[0]) {
+foreach ($data as $mimeType => $mimeTypeInformation) {
+    if (!array_key_exists('extensions', $mimeTypeInformation)) {
```

Comment:
`array_key_exists` is deprecated on PHP 7.4 shouldn't we use `isset` instead as it is available on all versions.?

Reply:
AFAIK, `array_key_exists` is not deprecated in 7.4 so it should be fine.

Reply:
I was looking for more info and it is deprecated to be used on objects, not for arrays. https://www.php.net/manual/en/migration74.deprecated.php#migration74.deprecated.core.array-key-exists-objects